browser/ui/webui/settings/brave_sync_handler.cc

```diff
@@ -241,3 +239,19 @@ base::Value BraveSyncHandler::GetSyncDeviceList() {
   }
   return device_list;
 }
+
+void BraveSyncHandler::OnCodeGeneratorResponse(
+    base::Value callback_id,
+    const qrcode_generator::mojom::GenerateQRCodeResponsePtr response) {
+  if (response->error_code !=
+      qrcode_generator::mojom::QRCodeGeneratorError::NONE) {
+    LOG(ERROR) << "QR code generated failure: " << response->error_code;
```

Comment:
generated -> generator 
and does this need to be ERROR? Seems like VLOG?

Reply:
fixed in https://github.com/brave/brave-core/pull/5926/commits/491460d8b232b428f9a9482263796a6932157238